Add index to board support activity (#651)
publiccodenet/about: activities/board-support/index.md

---
type: resource
---

# Board of directors support

> The work done to support the [board of directors](../../organization/governance-model.md#board-of-directors) with their meetings and tasks.

Reviewer: ```suggestion
This work supports the [board of directors](../../organization/governance-model.md#board-of-directors) with their meetings and tasks:
```
